tests/integration/records/relationship-changes-test.js

```diff
@@ -0,0 +1,110 @@
+import setupStore from 'dummy/tests/helpers/store';
+import Ember from 'ember';
+
+import {module, test} from 'qunit';
+
+import DS from 'ember-data';
+
+var env, store, Author, Post;
+var run = Ember.run;
+
+module('integration/records/relationship-changes - Relationship changes', {
+  beforeEach() {
+    Author = DS.Model.extend({
```

Comment:
Can we define these models at the module level, that way our tests don't have to constantly recreate these classes. This scenario likely doesn't affect the overall test run speed so much, but in aggregate they really do add up